examples/benchmark/tls_bench.c

```diff
@@ -383,6 +383,32 @@ char* myoptarg = NULL;
 int DoneHandShake = 0;
 #endif
 
+
+#if defined(HAVE_FIPS) && defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION == 5)
+static int run_all_CAST(void)
+{
+    int ret = 0;
+    int cast_idx = 0;
+
+    for (cast_idx=0; cast_idx<FIPS_CAST_COUNT; cast_idx++) {
```

Comment:
Should have spaces around '=' and '<'.

Reply:
@bigbrett please fix. Thanks

Reply:
fixed in latest commit. @dgarske thank you for bringing this PR back from the dead!!